src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -172,33 +172,69 @@ then you have to download the database driver and put it at: *PROACTIVE_HOME/add
 
 Scheduler, and 3 microservices (*scheduling-api*, *job-planner*, and *proactive-cloud-watch*) have to point to the same database.
 
-== Installation on a Cluster
+== Deploy ProActive Nodes
 
-Adding Compute Hosts of a cluster to the ProActive Scheduler typically involves unpacking the release archive on all those hosts.
-Once it's done you need to run a ProActive Node on the Compute Host and connect it to the ProActive Scheduler. There are two principal ways of
-doing that:
+In the ProActive Scheduler, all the computational resources (<<_glossary_compute_host,Compute Hosts>>) are managed as a set of <<_glossary_node_source,*Node Sources*>>.
```

Comment:
Computational Resources are managed by the ProActive Resource Manager component.

In the ProActive Resource Manager, ...